apps/src/tutorialExplorer/filterHeader.jsx

```diff
@@ -17,33 +17,24 @@ const styles = {
     paddingRight: 7,
     backgroundColor: 'white'
   },
-  sticky: {
-    boxShadow: "0 8px 6px -6px #999"
-  },
   barDesktop: {
     color: "dimgrey",
```

Comment:
Where are these colors coming from? I'm used to seeing something like `color.white` with a color import from utils.

Reply:
Good point.  Tutorial Explorer has never used the color import but it should get a separate update to use it everywhere.  Noting.